biojava-core/src/test/java/org/biojava/nbio/core/sequence/io/embl/EmblParserTest.java

```diff
@@ -0,0 +1,26 @@
+package org.biojava.nbio.core.sequence.io.embl;
+
+import org.junit.Test;
+
+import java.io.File;
+/**
+ * This class should test the parser of EmblParser class
+ * @author Noor Aldeen Al Mbaidin
+ */
+public class EmblParserTest {
+
+    @Test(expected = NullPointerException.class)
+    public void givenNullFileParameterWhenCreateEmblParserThenthrowException(){
+        File file = null;
+        EmblParser emblParser = new EmblParser(file);
+    }
+
+    @Test
+    public void test(){
```

Comment:
You will need to assert something in the test, e.g. whether fields are not null and checking that values read for the fields are as expected.

Reply:
sorry for that i didn't write full tests for the class but i'm working to write cases to cover more percentage of the code.